secure_message/repository/retriever.py

```diff
@@ -31,22 +31,40 @@ def unread_message_count(user):
         return result
 
     @staticmethod
-    def thread_count_by_survey(survey, is_closed):
+    def thread_count_by_survey(surveys, is_closed, user, my_conversations=False, ru_id=None, cc=None, ce=None):
```

Comment:
Take a look at https://github.com/ONSdigital/ras-secure-message/blob/master/secure_message/resources/threads.py#L108 .  I think this function should be changed to operate in a similar way where you pass in an object (in ThreadList it's whatever get_options returns) that contains all the bits as a single variable instead of passing them in 1 by 1.
Pylint was complaining that there were 7 variables being passed in, and while I could've ignored it, I agree that this function has too many variables being passed in.

Reply:
Added for consistency but feels a bit dirty, naming the specific params shows intent .